src/components/TileCatalog/TileCatalog.jsx

```diff
@@ -1,189 +1,176 @@
-              checked={selectedTileId === tile.id}
-              onChange={onSelection}
-            >
-              {tile.renderContent
-                ? tile.renderContent({ values: tile.values, id: tile.id })
-                : tile.value}
-            </RadioTile>
-          ))}
-        />
-      ) : (
-        <StyledEmptyTile>
-          {error || (
-            <Fragment>
-              <Bee32 />
-              <p>{(search && search.noMatchesFoundText) || 'No matches found'}</p>
-            </Fragment>
-          )}
-        </StyledEmptyTile>
-      )}
-      {!isLoading && tiles.length > 0 && !error && pagination ? (
-        <SimplePagination {...pagination} maxPage={Math.ceil(totalTiles / pageSize)} />
-      ) : null}
-    </StyledContainerDiv>
+    <div>
+      <div>
+        {/* {persistentSearch ? (
+          <div className="tile-catalog--persistent-search">
+            <Search placeHolderText="" onChange="'" size="sm" value="" labelText="" />
+          </div>
+        ) : null} */}
+        <div />
+      </div>
+
+      <div className={`${iotPrefix}--tile-catalog--canvas-container`}>
+        <div className={`${iotPrefix}--tile-catalog--tile-canvas`}>
+          {/* {featuredTile ? (
+            <div>
+              <div className="tile-catalog--tile-canvas--featured-tile-title">
+                {featuredTileTitle}
+              </div>
+              <div className="tile-catalog--tile-canvas--featured-tile">{featuredTile}</div>
+            </div>
+          ) : null} */}
+          <div className={`${iotPrefix}--tile-catalog--tile-canvas--header`}>
+            <div className={`${iotPrefix}--tile-catalog--tile-canvas--header--title`}> {title}</div>
+            {hasSearch ? (
+              <TableToolbarSearch
+                placeHolderText={i18n.searchPlaceHolderText}
+                onChange={onSearch}
+                className={`${iotPrefix}--tile-catalog--tile-canvas--header--search`}
+              />
+            ) : null}
+            <div className={`${iotPrefix}--tile-catalog--tile-canvas--header--select`}>
+              {hasSort ? (
+                <Select
+                  onChange={evt => onSort(evt.target.value)}
+                  defaultValue={selectedSortOption}
+                  labelText=""
+                >
+                  {sortOptions.map(option => (
+                    <SelectItem text={option.text} value={option.id} />
+                  ))}
+                </Select>
+              ) : null}
+            </div>
+          </div>
+
+          <div className={`${iotPrefix}--tile-catalog--tile-canvas--content`}>{renderGrid()}</div>
+          <div className={`${iotPrefix}--tile-catalog--tile-canvas--bottom`}>
+            <TilePagination
+              page={currentPage}
+              numPages={Math.ceil(tiles.length / (numRows * numColumns))}
+              onChange={newPage => setCurrentPage(newPage)}
+            />
+          </div>
+        </div>
+        {/* {hasFilter ? (
```

Comment:
Is this suppose to be removed?

Reply:
These are additions to the component coming soon, I'm fine with leaving them in as comments.